platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/MemoryIndexDiskKeyValueStore.java

```diff
@@ -229,11 +229,11 @@ public void resumeMerging() throws IOException {
      *
      * @throws IOException If there was a problem opening a writing session
      */
-    public void startWriting(final long minimumValidKey) throws IOException {
+    public void startWriting(final long minimumValidKey, final long maxValidIndex) throws IOException {
```

Comment:
Now both `startWriting()` and `endWriting()` take `minimumValidKey` and `maxValidIndex` as parameters. They must be the same between matching calls which adds unnecessary burden on the API. Can those be saved in `startWriting()` and reused in `endWriting()`?